File: 3636-check-balanced-string/check-balanced-string.py
class Solution(object):
    def isBalanced(self, num):
        sumeven=0
        sumodd=0
        for i in range(len(num)):
            if(i%2!=0):
                sumodd=sumodd+int(num[i])
            else:
                sumeven=sumeven+int(num[i])
        return sumeven==sumodd
        
        

File: 3636-check-balanced-string/check-balanced-string.cpp
class Solution {
public:
    bool isBalanced(string num) {
        int sumodd=0,sumeven=0;
        for(int i=0;i<num.length();i++){
            if(i%2!=0){
              sumodd=sumodd+(num[i]-'0');
            }
            else{
                sumeven=sumeven+(num[i]-'0');
        }}return sumodd==sumeven;
    }
};